pycds/climate_baseline_helpers.py

```diff
@@ -140,40 +155,58 @@ def parse_line(line):
         .filter(Variable.network.has(name=network_name))\
         .first()
     if not variable:
-        raise ValueError("Climate variable named '{}' associate with network {} was not found in the database"
+        raise ValueError("Climate variable named '{}' associated with network {} was not found in the database"
                          .format(var_name, network_name))
 
-    # TODO: Use logging rather than print throughout
-    print('Loading...')
+
+    if var_name in ['Tx_Climatology', 'Tn_Climatology']:
+        convert = lambda temp_in_10thsC: float(temp_in_10thsC) / 10
+    else:
+        convert = lambda precip_in_mm: float(precip_in_mm)
+
+
+    logging.info('Loading...')
     n_added = 0
+    n_excluded = 0
     n_skipped = 0
 
     for line in lines:
         data = parse_line(line)
-        latest_history = session.query(History)\
-            .filter(History.station.has(native_id=data['native_id']))\
-            .order_by(History.sdate.desc())\
-            .first()
-        if latest_history:
-            session.add_all(
-                [DerivedValue(
-                    time=datetime.datetime(baseline_year, month, baseline_day(month), baseline_hour),
-                    datum=float(data[str(month)]),
-                    vars_id=variable.id,
-                    history_id=latest_history.id
-                ) for month in range(1, 13)]
-            )
-            n_added += 1
+        station_native_id = data['native_id'].strip()
+        if station_native_id not in exclude:
+            latest_history = session.query(History)\
+                .filter(History.station.has(native_id=station_native_id))\
+                .order_by(History.sdate.desc())\
+                .first()
+            if latest_history:
+                logging.info('Adding station "{}"'.format(station_native_id))
+                for month in range(1, 13):
+                    datum = data[str(month)]
+                    if datum != '-9999':
+                        session.add(
+                            DerivedValue(
+                                time=datetime.datetime(baseline_year, month, baseline_day(month), baseline_hour),
+                                datum=convert(datum),
+                                variable=variable,
+                                history=latest_history
+                            )
+                        )
+                n_added += 1
+            else:
+                logging.info('Skipping input line:')
```

Comment:
Looks like good, informative logging overall. Thanks for making sure that the user knows what's going on.